configs/deblurganv2/deblurganv2_fpn-inception_1xb1_gopro.py

```diff
@@ -0,0 +1,193 @@
+        }, {
+            'name': 'rgb_shift'
+        }, {
+            'name': 'hsv_shift'
+        }, {
+            'name': 'sharpen'
+        }]),
+    dict(type='PackInputs')
+]
+
+test_pipeline = [
+    dict(type='LoadImageFromFile', key='img'),
+    dict(type='LoadImageFromFile', key='gt'),
+    dict(type='PackInputs')
+]
+
+data_root = 'G:/github/DeblurGANv2/data/gopro/debug/'
+
+train_dataloader = dict(
+    batch_size=1,
+    num_workers=4,
+    persistent_workers=False,
+    sampler=dict(type='DefaultSampler', shuffle=False),
+    dataset=dict(
+        type='BasicImageDataset',
+        metainfo=dict(dataset_type='gopro', task_name='deblur'),
+        data_root=data_root + 'train',
+        data_prefix=dict(img='input', gt='target'),
+        # ann_file='meta_info_gopro_train.txt',
+        pipeline=train_pipeline))
+
+val_dataloader = dict(
+    num_workers=4,
+    persistent_workers=False,
+    drop_last=False,
+    sampler=dict(type='DefaultSampler', shuffle=False),
+    dataset=dict(
+        type='BasicImageDataset',
+        metainfo=dict(dataset_type='gopro', task_name='deblur'),
+        data_root=data_root + 'test',
+        # ann_file='meta_info_gopro_test.txt',
+        data_prefix=dict(img='input', gt='target'),
+        pipeline=val_pipeline))
+
+test_dataloader = dict(
+    num_workers=4,
+    persistent_workers=False,
+    drop_last=False,
+    sampler=dict(type='DefaultSampler', shuffle=False),
+    dataset=dict(
+        type='BasicImageDataset',
+        metainfo=dict(dataset_type='gopro', task_name='deblur'),
+        data_root=data_root + 'test',
+        # ann_file='meta_info_gopro_test.txt',
+        data_prefix=dict(img='input', gt='target'),
+        pipeline=test_pipeline))
+
+val_evaluator = dict(
+    type='Evaluator', metrics=[
+        dict(type='PSNR'),
+        dict(type='SSIM'),
+    ])
+
+train_cfg = dict(type='EpochBasedTrainLoop', max_epochs=100)
+val_cfg = dict(type='MultiValLoop')
+test_cfg = dict(type='MultiTestLoop')
+test_evaluator = val_evaluator
+
+# optimizer
+optim_wrapper = dict(
+    constructor='MultiOptimWrapperConstructor',
+    generator=dict(
+        type='OptimWrapper',
+        optimizer=dict(type='Adam', lr=0.0001, betas=(0.5, 0.999))),
+    discriminator=dict(
+        type='OptimWrapper',
+        optimizer=dict(type='Adam', lr=0.0001, betas=(0.5, 0.999))),
+)
+
+# learning policy
+param_scheduler = dict(
+    type='LinearLR',
+    start_factor=0.0001,
+    end_factor=0.0000001,
+    begin=50,
+)
+
+default_hooks = dict(
+    checkpoint=dict(
+        type='CheckpointHook',
+        interval=1000,
+        save_optimizer=True,
+        by_epoch=False,
+        out_dir=save_dir,
+        save_best='auto',
+    ),
+    timer=dict(type='IterTimerHook'),
+    logger=dict(type='LoggerHook', interval=1),
+    param_scheduler=dict(type='ParamSchedulerHook'),
+    sampler_seed=dict(type='DistSamplerSeedHook'),
+)
+
+load_from = 'G:/github/DeblurGANv2/fpn_inception1.pth'
```

Comment:
it would be better to use a web link to the model weights instead of a local path.